base/sparse/umfpack.jl

```diff
@@ -113,12 +148,13 @@ function lufact{Tv<:UMFVTypes,Ti<:UMFITypes}(S::SparseMatrixCSC{Tv,Ti})
     finalizer(res, umfpack_free_symbolic)
     umfpack_numeric!(res)
 end
-lufact(A::SparseMatrixCSC) = lufact(float(A))
+lufact{T<:Real}(A::SparseMatrixCSC{T}) = lufact(map(Float64, A))
```

Comment:
Our current deprecation warning says to use `map(Float64, A)`. Is that worth changing?